### ISSUE TITLE
There is no way to select document using Retyped.jQuery.

### ISSUE THREAD

**Issue by hardhub** [Retyped/Retyped#61]
$(document); // This is not possible using C# and Retyped.  

Bridge.Html5: jQuery.Document  
Retyped: jQuery.select((TypeOrArray<Element>)document.documentElement)  

And these lines are not equivalent (though suitable for most cases).  

So it does not look enough elegant as it is for native jquery or for Bridge.Html5.  
I think verbosity of Retyped is a big problem.  
Code becomes not clear.

**Comment by geoffreymcgill** [comment]
The following `jQuery.select(document);` sample using **Retyped.jquery** appears to work well and is closer to the intent of jQuery than using `jQuery.Document`.

https://deck.net/46d7a69e8bb8b3fca52fd6a0d316524b

```csharp
using static Retyped.jquery;
using static Retyped.dom;

public class Program
{
    public static void Main()
    {
        jQuery.select(document);
    }
}
```

Hope this helps.

**Comment by hardhub** [comment]
> appears to work

It does not look so:

![image](https://user-images.githubusercontent.com/4306334/40023206-f8e92868-57d2-11e8-8160-201cfad9057f.png)

**Comment by geoffreymcgill** [comment]
It appears the source jquery .d.ts has changed between releases and with the current Retyped logic has prevented the direct use of the `document` object. 

We have reviewed in detail and there is a plan in place to fix, which is likely to happen this week.

Once the new **Retyped.jquery** build is published, the `jQuery.select(document);` sample above will work as expected.

**Comment by AndreyChechel** [comment]
Hi @hardhub,

We've implemented a few enhancements in **Retyped.jquery**. 
The updated package is available on NuGet:
[Retyped.jquery 3.3.6711](https://www.nuget.org/packages/Retyped.jquery/3.3.6711)

It also includes fixes for #56 and #57. Hope you'll find them useful.

**Comment by geoffreymcgill** [comment]
The `jQuery.select(document)` sample provided above is now working with the latest **Retyped.jquery** release.

https://deck.net/46d7a69e8bb8b3fca52fd6a0d316524b
